### ISSUE TITLE
Add PS1 style music settings

### ISSUE THREAD

**Issue by Richard-L** [LostArtefacts/TRX#1626]
TR2Main:

![AudioSelected](https://github.com/user-attachments/assets/1e78fbc6-0a28-4598-8539-872f94b37d99)

Curiously the PS1 is missing its menu background here:

![SLUS-00437_2023-11-05_23-58-22](https://github.com/LostArtefacts/TRX/assets/4934209/3a8c61b3-1306-4c8c-a110-c624a3087077)

**Comment by Richard-L** [comment]
First screenshot above is now resurrected.

**Comment by walkawayy** [comment]
@Richard-L is this completed? I think we have this option and more available now. There's no before screenshots to compare against, so idk what exactly you were asking.

**Comment by Richard-L** [comment]
I meant the gradient bar style.

Before it were just numbers from 1 to X. Now I think we have %.

In principle I still think the bars would be nice, but I suppose with the wealth of settings all brought together, it would feel off to cram bars into the menu still. So maybe this is something that just can't be...

Perhaps dash you could share your thoughts when you read this.

**Comment by rr-** [comment]
Yeah – it'll look off. I'll keep it on my mind but it's unlikely we'll be doing something about it.
